libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -24,6 +24,12 @@ import type { SelectionParameterSet } from '../parsing/SelectionParameterSet';
 import type { RushConfigurationProject } from '../../api/RushConfigurationProject';
 import type { Subspace } from '../../api/Subspace';
 
+interface ISubspaceSelectedProjectsMetadata {
+  selectedProjects: Set<RushConfigurationProject>;
+  pnpmFilterArguments: string[];
+  alwaysFullInstall: boolean;
```

Comment:
Is `alwaysFullInstall` actually used?  I see it being assigned but never read from.  Are you sure we need to store it in this interface?

Reply:
It's used on line 178.